### PR TITLE
Most read page

### LINKED ISSUE #4472: Parent Issue: Most Read as a user-facing URL
Currently all WS sites have a Most Read page as a URL users can visit. The URL is of the format:
`bbc.com/<service>/popular/read` eg. /igbo/popular/read

If we want to maintain this url, and I think at present we do, then we will need to recreate this page somewhere, for all services, and it seems making use of the new most read component in a page is logical.

**Describe the solution you'd like**
I would like a page in Simorgh to be available for each service of this format:
`bbc.com/<service>/popular/read`
If a user visits that page I would like them to be able to see a list of popular news stories from that service.
Accessibility - Most read should be within the main container and the h1 should be visually displayed as ‘Most read’ and the skip to content link should anchor here (note there should not be a h2 for most read or any other regions for most read within the main region) 

The outcome of this will be that we have a new page type and route in Simorgh, which will initially be available locally, and we can then get feedback from UX and make changes accordingly.

**Describe alternatives you've considered**
In theory, in a world in which we will be building other most popular lists of content there could be an argument that using the url `bbc.com/<service>/popular` might be better, since it would allow us to stack Most Read, Most Watched, Most popular topics etc in a single page.
But that would then require a bunch of redirects from /popular/read to /popular

**Testing notes**
To be tested according to design doc across various breakpoints.Also the ATI and chartbeat testing for this new page.

- Unit tests
- Integration tests
- One cypress test to cover this as it is a new page type.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.
**Additional context**
This piece of work is potentially a thing we could do, or not do. Before we get deep into it I'd be interested in an estimate of how much work is needed to make this happen for all sites.

### PR DESCRIPTION
**Pull request by thekp**
Resolves #4472

**Overall change:**
_Create most read page that is using server side fetched data. Refactored most read to accept a wrapper prop so that each page pass it's own wrapper._

To see locally: 
- http://localhost:7080/pidgin/articles/cwl08rd38l6o 
- http://localhost:7080/pidgin/popular/read
- http://localhost:7080/pidgin

**Code changes:**

- _Updated getInitialsData function to pass an object that contains a `path` and `service`, we done this so that we can use the service value to create the url for the most read data endpoint_
- _Added most read page, we used the h1 specs specified by UX._
- _Refactored most read, so it accepts a `wrapper` prop, which allows us to pass a wrapper for most read on each page type. For example on front pages we pass in a wrapper that contains a section label and desired wrapper for the front page most read requirements._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -0,0 +1,1144 @@
...
+    <meta
+      content="De one we dem de read well well - BBC News Pidgin"
+      name="twitter:title"
+    />
+  </head>
+  <body>
+    <div>
+      <div>
+        chartbeat
+      </div>
```
Comment (greenc05) [inline src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap:552]: Should the banner/navigation be on here?

File: src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -0,0 +1,1144 @@
...
+      <main
+        class="c0"
+        dir="ltr"
+        role="main"
+      >
+        <h1
+          class="c1"
+          id="content"
```
Comment (greenc05) [inline src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap]: Missing tabindex attribute?

Comment (thekp) [reply src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap]: Thanks for looking into this Catharine, I have fixed this now :)

File: src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (thekp) [reply src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap:552]: I had a look at the other page snapshots and I couldn't see navigation/banner/footer in these.

**Comment by tochwill** [comment]
Wondering, should we split this into 2 PRs, one for the changes to getInitialData that can be regression tested, and then one that adds the most read page in on top of it? Might be faster to review! Will bring up in stand up.

**Comment by thekp** [comment]
I have split this PR into two separate PRs as suggested above ^

PR1 (Update getInitialData): https://github.com/bbc/simorgh/issues/6169
PR2 (Add Page and container refactor): https://github.com/bbc/simorgh/pull/6175

**Pull request closed without merging**